packages/hash/api/src/graph/workspace-types.ts

```diff
@@ -95,7 +94,7 @@ export const ensureWorkspaceTypesExist = async (params: {
       if (!existingEntityType) {
         await EntityTypeModel.create(graphApi, {
           accountId: workspaceAccountId,
-          schema,
+          schema: schema as any, // TODO - trust me bro
```

Comment:
@thehabbos007 are we okay with the `any` conversion for now or do we want to do something else here?

(Either way going to replace this todo with something sensible)

Reply:
As we have the type system types in place now, I'd change the `create` static method's signature to make use of that instead as we did for `DataTypeModel` and `PropertyTypeModel` [here](https://github.com/hashintel/hash/pull/977/files#diff-99c75fb2d60e4a08d67c29daf76c7155b5316d11bc2164bbaba8b4dd96a1ed09).

Reply:
That should be done, thanks for the help